src/components/pageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography, useMediaType, useTranslation } from '@okp4/ui'
+import type { UseTranslationResponse } from '@okp4/ui'
+import './pageTitle.scss'
+
+type PageTitleProps = {
+  readonly title: string
+}
+
+const PageTitle = ({ title }: PageTitleProps): JSX.Element => {
+  const { t }: UseTranslationResponse = useTranslation()
+  const isSmallScreen: boolean = useMediaType('(max-width: 995px)')
```

Comment:
Why do you not use the breakpoints declared in the `useBreakpoint` hook?
We should follow this convention if we want to upgrade the layout to a 12 columns grid in a next future.

Reply:
I used this breakpoint in order to use the same breakpoint used in the `DataspaceSummary` component for a medium screen.

Reply:
OK
I didn't notice that in the last PR... 
This is not a good practice as it does not fit our breakpoints exported by the library, so the layout will be inconsistent...
We need to fix this in a dedicated PR, thanks